inference-engine/src/legacy_api/src/ie_cnn_layer_builder_ngraph.cpp

```diff
@@ -2109,9 +2109,21 @@ CNNLayer::Ptr NodeConverter<ngraph::op::NonMaxSuppressionIE>::createLayer(const
 
     auto res = std::make_shared<InferenceEngine::NonMaxSuppressionLayer>(params);
     res->params["sort_result_descending"] = std::to_string(castedLayer->m_sort_result_descending);
-    res->params["center_point_box"] = std::to_string(castedLayer->m_sort_result_descending);
+    res->params["center_point_box"] = std::to_string(castedLayer->m_center_point_box);
     return res;
 }
 
+template <>
+CNNLayer::Ptr NodeConverter<ngraph::op::NonMaxSuppressionIE2>::createLayer(const std::shared_ptr<ngraph::Node>& layer) const {
```

Comment:
Please use visitors API and specific creators to convert node to CNNLayer

Reply:
Done